src/app/components/operations/login/login.component.ts

```diff
@@ -0,0 +1,85 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { LoginUser } from 'src/app/models/login-user';
+import { AuthService } from 'src/app/services/auth.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7122565)
```suggestion
import { AuthService } from "src/app/services/auth.service";
```